### PR TITLE
fix issue 433

### PR DESCRIPTION
**Pull request by PhilCai1993**
## Pull request checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by PhilCai1993** [comment]
#433

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ This release closes the [2.2.0 milestone](https://github.com/Instagram/IGListKit
 - Fix bug where emptyView's hidden status is not updated after the number of items is changed with `insertInSectionController:atIndexes:` or related methods. [Peter Edmonston](https://github.com/edmonston) [(#395)](https://github.com/Instagram/IGListKit/pull/395)
 - Fix bug where `IGListStackedSectionController`'s children need to know `numberOrItems` before didUpdate is called. [(#348)](https://github.com/Instagram/IGListKit/pull/390)
 - Fix bug where `-[UICollectionViewCell ig_setStackedSectionControllerIndex:]` should use `OBJC_ASSOCIATION_COPY_NONATOMIC` for NSNumber. [PhilCai](https://github.com/PhilCai1993) [(#424)](https://github.com/Instagram/IGListKit/pull/426)
+- Rename property `sectionControllerToObjectMap`  to `objectToSectionControllerMap` in `IGSectionMap`. [PhilCai](https://github.com/PhilCai1993) [(#433)](https://github.com/Instagram/IGListKit/pull/436)
```
Comment (rnystrom) [inline CHANGELOG.md]: @jessesquires should we add a changelog entry for a private/internal structure?

Comment (PhilCai1993) [inline CHANGELOG.md]: If it's not needed, I will delete this😄

Comment (Sherlouk) [inline CHANGELOG.md]: @rnystrom My two cents would not to have entries for structural changes unless it fixes a bug

**Comment by PhilCai1993** [comment]
@rnystrom I deleted unnecessary changelog entry 😊

**Pull request closed without merging**